visualization/app/codeCharta/state/effects/updateFileSettings/utils/attributeDescriptors.merger.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { AttributeDescriptors } from "../../../../codeCharta.model"
+import { getMergedAttributeDescriptors } from "./attributeDescriptors.merger"
+
+describe("AttributeDescriptorsMerger", () => {
+	describe("getMergedAttributeDescriptors", () => {
+		let attributes1: AttributeDescriptors
+		let attributes2: AttributeDescriptors
+		let attributes3: AttributeDescriptors
+		let attributes4: AttributeDescriptors
+
+		beforeEach(() => {
+			attributes1 = undefined
+
+			attributes2 = {}
+
+			attributes3 = {
+				rloc: {
+					title: "rloc3",
+					description: "",
+					hintLowValue: "",
+					hintHighValue: "",
+					link: "https://www.npmjs.com/package/metric-gardener"
+				}
+			}
+
+			attributes4 = {
+				rloc: {
+					title: "rloc4",
+					description: "DescriptionRloc4",
+					hintLowValue: "",
+					hintHighValue: "",
+					link: "https://www.npmjs.com/package/metric-gardener"
+				},
+				comment_lines: {
+					title: "Comment Lines",
+					description: "Number of lines containing either comment or commented-out code",
+					hintLowValue: "",
+					hintHighValue: "",
+					link: "https://www.npmjs.com/package/metric-gardener"
+				}
+			}
+		})
+
+		it("should skip undefined descriptors (do we really need this edge case?)", () => {
```

Comment:
We might night it for old maps, where we do not have any descriptors yet?
Let's remove the comment in the testname. I would rather add a TODO in the code instead of the test name.